app/router.js

```diff
@@ -1,9 +1,21 @@
 import Ember from 'ember';
 import config from './config/environment';
 
+// Disable the reset scroll behaviour on transitions between the two given routes.
+const SCROLL_RESET_DISABLED_ROUTES = ['preprints.provider.moderation', 'preprints.provider.settings'];
+
 const Router = Ember.Router.extend({
     location: config.locationType,
-    rootURL: config.rootURL
+    rootURL: config.rootURL,
+    willTransition(oldInfos, newInfos) {
+        this._super(...arguments);
+        const prevPath = Ember.Router._routePath(oldInfos);  //Previous route
+        const newPath = Ember.Router._routePath(newInfos);   //New route
```

Comment:
You can remove these two comments, I think it's pretty clear what these are. If you want, could rename the variables to `oldRoute` and `newRoute` or something like that.